lib/plurimath/math/formula.rb

```diff
@@ -43,6 +43,70 @@ def to_html
         value.map(&:to_html).join
       end
 
+      def to_omml
+        attributes = {
+          "xmlns:m": "http://schemas.openxmlformats.org/officeDocument/2006/math",
+          "xmlns:mc": "http://schemas.openxmlformats.org/markup-compatibility/2006",
+          "xmlns:mo": "http://schemas.microsoft.com/office/mac/office/2008/main",
+          "xmlns:mv": "urn:schemas-microsoft-com:mac:vml",
+          "xmlns:o": "urn:schemas-microsoft-com:office:office",
+          "xmlns:r": "http://schemas.openxmlformats.org/officeDocument/2006/relationships",
+          "xmlns:v": "urn:schemas-microsoft-com:vml",
+          "xmlns:w": "http://schemas.openxmlformats.org/wordprocessingml/2006/main",
+          "xmlns:w10": "urn:schemas-microsoft-com:office:word",
+          "xmlns:w14": "http://schemas.microsoft.com/office/word/2010/wordml",
+          "xmlns:w15": "http://schemas.microsoft.com/office/word/2012/wordml",
+          "xmlns:wne": "http://schemas.microsoft.com/office/word/2006/wordml",
+          "xmlns:wp": "http://schemas.openxmlformats.org/drawingml/2006/wordprocessingDrawing",
+          "xmlns:wp14": "http://schemas.microsoft.com/office/word/2010/wordprocessingDrawing",
+          "xmlns:wpc": "http://schemas.microsoft.com/office/word/2010/wordprocessingCanvas",
+          "xmlns:wpg": "http://schemas.microsoft.com/office/word/2010/wordprocessingGroup",
+          "xmlns:wpi": "http://schemas.microsoft.com/office/word/2010/wordprocessingInk",
+          "xmlns:wps": "http://schemas.microsoft.com/office/word/2010/wordprocessingShape",
+        }
+        para_element = Utility.omml_element("m:oMathPara", attributes)
+        math_element = Utility.omml_element("m:oMath")
```

Comment:
Don't you need `attributes`?

Reply:
If you're calling Utility.omml_element with attributes every time, perhaps you should have a builder class that just supplies the `attributes` initially, e.g.

```ruby
builder = Utility::Builder.new(type: :omml, attributes: attributes)
builder.element('m:oMathPara')
# or, builder.element('oMathPara', namespace: 'm')
```

Reply:
> If you're calling Utility.omml_element with attributes every time, perhaps you should have a builder class that just supplies the `attributes` initially, e.g.
> 
> ```ruby
> builder = Utility::Builder.new(type: :omml, attributes: attributes)
> para_element = builder.element('m:oMathPara')
> # or, builder.element('oMathPara', namespace: 'm')
> ```

We are not passing attributes for every element(OMML tag).
The ones that are passed contains different attributes for each element.
Not sure how builder class can help in this case. Can you explain the flow for builder class?

Reply:
Ah okay